imgui-SFML.cpp

```diff
@@ -987,8 +990,8 @@ void Image(const sf::Sprite& sprite, const sf::Vector2f& size, const sf::Color&
     }
 
     const sf::Texture& texture = *texturePtr;
-    sf::Vector2f textureSize = static_cast<sf::Vector2f>(texture.getSize());
-    const sf::IntRect& textureRect = sprite.getTextureRect();
+    const sf::Vector2f textureSize{texture.getSize()};
+    const sf::FloatRect textureRect{sprite.getTextureRect()};
```

Comment:
```suggestion
    const sf::Vector2f textureSize(texture.getSize());
    const sf::FloatRect textureRect(sprite.getTextureRect());
```
Let's keep the style consistent and keep using parenthesis to call constructors.